README.md

```diff
@@ -45,12 +45,41 @@ Check out [Hyperledger Iroha 2 Tutorial](https://hyperledger.github.io/iroha-2-d
 
 ## Maintenance
 
-### Explore Jake tasks
+Make sure you have installed Node.js v16.17 or v18. As for a package manager, this project uses [PNPM](https://pnpm.io/).
```

Comment:
not v17?

Reply:
Odd versions of Node.js are unstable, even - long-term supported.

In general, it is not very important to mention a lot of versions here - v16.16 is enough.

Reply:
ok